vm/src/obj/objdict.rs

```diff
@@ -495,6 +495,22 @@ macro_rules! dict_iterator {
             fn len(&self, vm: &VirtualMachine) -> usize {
                 self.dict.clone().len(vm)
             }
+
+            #[pymethod(name = "__repr__")]
+            #[allow(clippy::redundant_closure_call)]
```

Comment:
which part raises this warning?

Reply:
It is the use of `result_fn` same as in `next`